docs/finality-provider-phase2.md

```diff
@@ -0,0 +1,589 @@
+You will need this backup file to import the key into the Phase 2 environment.
+
+### Step 2: Import Your EOTS Key into the Keyring
+To load your existing EOTS key, use the following command to import it into the 
+keyring:
+
+```shell 
+eotsd keys import <key-name> <path-to-backup>
+```
+
+- `<key-name>`: The name you want to assign to this key in Phase 2.
+- `<path-to-backup>`: The path to your EOTS key backup file from Phase 1.
+
+## Starting the EOTS Daemon
+
+You will need to navigate to the You can start the EOTS daemon using the following command:
+
+```shell 
+eotsd start --home <path> 
+```
+
+This command starts the EOTS RPC server at the address specified in eotsd.conf 
+under the RpcListener field (default: 127.0.0.1:12582). You can override this value 
+by specifying a custom address with the --rpc-listener flag.
+
+```shell
+2024-10-30T12:42:29.393259Z     info    Metrics server is starting
+{"addr": "127.0.0.1:2113"} 
+2024-10-30T12:42:29.393278Z     info    RPC server listening    {"address": "127.0.0.1:12582"} 
+2024-10-30T12:42:29.393363Z     info    EOTS Manager Daemon is fully active!  
+EOTS Manager Daemon is fully active!  
+```
+
+>**Note**: It is recommended to run the `eotsd` daemon on a separate machine or
+network segment to enhance security. This helps isolate the key management
+functionality and reduces the potential attack surface. You can edit
+the`EOTSManagerAddress` in the configuration file of the finality provider to
+reference the address of the machine where `eotsd` is running.
+
+## Setting up the Finality Provider
+
+The Finality Provider Daemon (FPD) is responsible for monitoring new Babylon blocks, 
+committing public randomness for the blocks it intends to provide finality signatures for, 
+and submitting finality signatures. For more information on Finality Providers, please see 
+[here](#).
+
+### Step 1: Initialize the Finality Provider Daemon
+
+Use the fpd init command to initialize a home directory for the Finality Provider. 
+You can set or change the home directory using the `--home` tag. For example, use 
+`--home ./fpKeys` to specify a custom directory.
+
+```shell 
+fpd init --home <path> 
+```
+
+>Note: Running this command may return the message 
+`service injective.evm.v1beta1.Msg does not have cosmos.msg.v1.service proto annotation`, 
+which is expected and can be ignored.
+
+### Step 2: Add key for the Finality Provider on the Babylon Chain
+
+The keyring is maintained by the finality provider daemon, this is local storage
+of the keys that the daemon uses. The account associated with this key exists on
+the babylon chain.
+
+Use the following command to add a key for your finality provider:
+
+```shell 
+fpd keys add --keyname <key-name> --keyring-backend test --home <path>
+```
+
+We use `--keyring-backend test`, which specifies which backend to use for the
+keyring, `test` stores keys unencrypted on disk. For production environments,
+use `file` or `os` backend.
```

Comment:
Not sure whether this will work for the finality provider daemon. As the finality provider will need to automatically and repeatedly send transactions, the key would have to be stored unencrypted on disk to enable access without having to paste the password each time. @gitferry can you verify?

Reply:
Yep, `test` indeed works, but what about `file` and `os`?